src/steps/Microsoft.DotNet.UpgradeAssistant.Steps.ProjectFormat/TryConvertTool.cs

```diff
@@ -42,9 +42,21 @@ public class TryConvertTool : ITryConvertTool
             }
 
             Path = tryConvertOptionsAccessor.Value.TryConvertPath;
+            MSBuildPath = HandleEscapeCharacters(tryConvertOptionsAccessor.Value.MSBuildPath);
             Version = GetVersion();
         }
 
+        /// <summary>
+        /// Path to MSBuild is often located in "Program Files" folder. We wrap the path with quotes
+        /// to handle the space as continuous argument and must also handle the backslash escape characters.
+        /// </summary>
+        private static string HandleEscapeCharacters(string folderPath)
+        {
+            return folderPath.Replace("\\", "\\\\");
```

Comment:
It's not clear to me why these need escaped. We don't need C:\\foo\\etc when we pass paths to try-convert from the command line, do we?

Reply:
The path to MSBUILD is `C:\Program Files\dotnet\sdk\5.0.202\` so we needed to add quotes

The escape chars prevent a sql injection style formatting problem with the TryConvertArgumentsFormat string

"--no-backup -m \"{0}\" --force-web-conversion --keep-current-tfms -p \"{1}\""